sandbox/codegen/__tests__/frameworks/solid.styled-factory.test.tsx

```diff
@@ -163,6 +163,24 @@ describe('styled factory - cva', () => {
       </button>
     `)
   })
+
+  test('html props', () => {
+    const { container } = render(
+      <styled.div htmlWidth={123} height="123">
+        Click me
+      </styled.div>,
+    )
+
+    expect(container.firstChild).toMatchInlineSnapshot(`
+      <div
+        class="h_123"
+        htmlwidth="123"
+        width="123"
```

Comment:
there is some side-effect here, only `width` is expected

Reply:
noticed that as well, note sure this can be fixed as the `mergeProps` is from solid, not from us, it doesn't split keys, it includes them in each matching function

maybe @segunadebayo knows ?